src/components/molecules/NavSearchBar/NavSearchBar.scss

```diff
@@ -16,60 +22,42 @@
       opacity: 0.8;
     }
   }
-}
-
-.nav__search input {
-  width: 18.125rem;
-  text-align: left;
-  background-color: $saturated-black;
-  border: solid 2px rgba(0, 0, 0, 0.13);
-  font-size: 0.9em;
-}
-
-.nav-search-results {
-  z-index: z(navbar-nav-search-results);
-
-  position: fixed;
-  top: 60px;
-
-  min-width: 18.125rem;
-
-  height: auto;
-  max-height: 75vh;
-  overflow-y: auto;
-
-  padding: 20px;
-  box-shadow: 0 10px 30px 0 rgba(0, 0, 0, 0.5);
-  border-radius: 0 0 24px 24px;
-  transition: transform 400ms cubic-bezier(0.23, 1, 0.32, 1);
-  background-color: $saturated-black;
-
-  .nav-search-result-number {
-    opacity: 0.5;
-    font-size: 0.9rem;
-  }
 
-  .row {
-    padding: 10px;
-    flex-direction: row;
-
-    &:hover {
-      background-color: rgb(80, 74, 87);
+  &__nav-dropdown {
+    position: fixed;
+    top: 0;
+    width: $search-results-width;
+    max-height: $search-results-height--max;
+    margin-top: $navbar-height;
+    background-color: $saturated-black;
+    box-shadow: box-shadow--large();
+    border-radius: 0 0 $border-radius--input $border-radius--input;
+    transform: translateY(-$navbar-height - $search-results-height--empty);
+    transform-origin: top right;
+    transition: all 400ms $transition-function;
+    pointer-events: none;
+
+    &--show {
+      transform: translateY(0);
+      opacity: 1;
+      pointer-events: all;
     }
 
-    .result-avatar {
-      width: 50px;
-      height: 50px;
-      border-radius: 5px;
-      margin-right: 10px;
-      background-position: center;
-      background-size: cover;
+    &__title {
+      margin-top: 0.5rem;
+      margin-bottom: 0.5rem;
+      padding: 0 $search-result-padding--horizontal;
+      opacity: 0.5;
     }
+  }
 
-    .result-info {
-      display: flex;
-      flex-direction: column;
-      justify-content: center;
-    }
+  &__search-results-number {
+    font-weight: 500;
+  }
+
+  &__search-results {
+    @include scrollbar;
+    overflow-y: auto;
+    max-height: 385px;
```

Comment:
Should this be using a variable from `navSearchBarConstants.scss`?

Reply:
added the constant: https://github.com/sparkletown/sparkle/pull/1328/commits/c7104e118d01c1d1d5100c9cf01154be7115c42e